mne/viz/_3d.py

```diff
@@ -666,8 +856,8 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
 
     for key in surfs.keys():
         surfs[key] = transform_surface_to(surfs[key], coord_frame, mri_trans)
-    src_rr = apply_trans(mri_trans, src_rr)
-    src_nn = apply_trans(mri_trans, src_nn, move=False)
+    # src_rr = apply_trans(mri_trans, src_rr)
+    # src_nn = apply_trans(mri_trans, src_nn, move=False)
```

Comment:
indeed. I've also seen weird alignments on CTF system see the SPM plot:

![spm_plot_trans](https://user-images.githubusercontent.com/161052/28313467-38bff790-6bb7-11e7-8909-e8bffed98c59.png)

helmets seems to high. Although the source reconstruction is correct.